aries-site/src/components/content/ContentCard.js

```diff
@@ -1,8 +1,12 @@
 import React, { forwardRef } from 'react';
 import PropTypes from 'prop-types';
-import { Box, Image, Text } from 'grommet';
+import { Box, Image, Text, defaultProps } from 'grommet';
 import { Identifier, Tile } from 'aries-core';
 
+const { size } = defaultProps.theme.global;
+const maxCardWidth = `${parseInt(size.medium, 10) +
+  parseInt(size.small, 10)}px`;
```

Comment:
Curious to hear the rationale behind this formula?

Reply:
Not having reviewed any other code but just picking up this thread here, I would have expected the card width to be controlled by a parent Grid and its columns definition.

Reply:
That's how it was prior. Was running into some awkward presentation in screen widths in between medium and small. The cards get awkwardly wide. Can/will remove for now.

Reply:
> Curious to hear the rationale behind this formula?

There is an awkward phase where the Grid column gets quite wide when switching from 2 columns to 1. Was trying to make it less dramatic.

![Screen Shot 2020-03-25 at 7 11 01 PM](https://user-images.githubusercontent.com/1756948/77600139-89a69180-6ecc-11ea-92d9-fbc2502b51d6.png)

Reply:
Removed.